src/osm_callbacks.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+#include "osm_callbacks.h"
+
+#include <fmgr.h>
+
+#define OSM_CALLBACKS_VAR_NAME "osm_callbacks"
+
+OsmCallbacks *
+ts_get_osm_callbacks(void)
+{
+	OsmCallbacks **ptr = (OsmCallbacks **) find_rendezvous_variable(OSM_CALLBACKS_VAR_NAME);
```

Comment:
i think we should have a static variable holding the result to avoid doing the lookup on every call

Reply:
find rendezvous variable already allocs the var as a hash entry and returns that on subsequent calls. So I don't think there is a need for an additional static here.